src/overlays/actors/ovl_End_Title/z_end_title_gfx.c

```diff
@@ -0,0 +1,480 @@
+    0xDEBE0E6E, 0xFEAD0C0C, 0x0DADFD6C, 0x0A090AAD, 0xFECFCFFF, 0xAF0E0EAF, 0xFFFE3D0A, 0x06020000, 0x00000000,
+    0x00000000, 0x00000307, 0x0BADFECE, 0xAE7D0D0E, 0xDFFFDFFF, 0x3F0F0FAF, 0xFF6E0E0D, 0x0D0D0D1E, 0xAEFFFF9F,
+    0xAFFF6F0E, 0x0E0E6EDE, 0x0E1EEEFF, 0xEEAE0D0D, 0x6DFEAE0D, 0x0DAEFE6E, 0x0E0E0E6E, 0xFEAE0D0C, 0x0DCEFD4C,
+    0x09090AAD, 0xFE6F0FAF, 0xFF6E0E1E, 0xFEDE0C08, 0x04010000, 0x00000000, 0x00000000, 0x00000206, 0x0AADFE6D,
+    0x0C0B0C0D, 0xDEFF6FFF, 0xDF0F0FAF, 0xFF6E0E0D, 0x0D0D0D0E, 0x0E6FFFAF, 0xAFFF6F0E, 0x0E0E6EDE, 0x0D0D4DFE,
+    0xFEAD0C0C, 0x6DFEAD0D, 0x0DAEFE6E, 0x0E0E0E6E, 0xFEAE0D0D, 0x1DFDFD0B, 0x09080AAD, 0xFE6F0F7F, 0xFEAE0E0E,
+    0xFEDD0B07, 0x03010000, 0x00000000, 0x00000000, 0x00000206, 0x0AACFD6C, 0x0A09090B, 0xDDFE0ECE, 0xFE8E0EAF,
+    0xFF6E0E3D, 0x4DCD6D0E, 0x0E9EFE9F, 0xAFFF6E0E, 0x3E6E6DDD, 0x0C0B0C9D, 0xFDAD0C0C, 0x6DFDAD0C, 0x0CADFE6E,
+    0x0E3E4E6E, 0xFEAE0E4D, 0xEDFD7C09, 0x070709AC, 0xFE6E1EEE, 0xFE3D0D0D, 0xFDDC0A06, 0x02000000, 0x00000000,
+    0x00000000, 0x00000205, 0x098BDC4A, 0x08060709, 0xBCDD0D0D, 0x9DFEBE9E, 0xDEDDDDDD, 0xBD7DDDFD, 0xEDEDAD0D,
+    0x8DDDDDDD, 0xDDDD4DBC, 0x0A09090B, 0xCB8B0A0A, 0x4BDC8C0B, 0x0B8CDDDD, 0xDDDDBD4D, 0xDDDDDDDD, 0xBC3B0A07,
+    0x0505088B, 0xDDDDDDDD, 0x6C0B0B0C, 0xDCBB0905, 0x02000000, 0x00000000, 0x00000000, 0x00000103, 0x06080908,
+    0x05040407, 0x090A0A0A, 0x0B1C8C3C, 0x0C0C0B0B, 0x0B0B0B0B, 0x0B0B0B0B, 0x0B0B0B0B, 0x0B0B0A09, 0x07060607,
+    0x09080808, 0x09090908, 0x090A0B0B, 0x0B0B0B0B, 0x0B0B0B0B, 0x0A080604, 0x03030609, 0x0A0B0B0A, 0x09080809,
+    0x09090603, 0x01000000, 0x00000000, 0x00000000, 0x00000002, 0x03050504, 0x03020204, 0x05060606, 0x07080809,
+    0x08080807, 0x07070707, 0x07070707, 0x07070707, 0x07070706, 0x04030304, 0x05050404, 0x05060505, 0x05060707,
+    0x07070707, 0x07070707, 0x06040302, 0x01020305, 0x06070706, 0x05040405, 0x05050302, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x01020202, 0x01010101, 0x02020203, 0x03040404, 0x04040303, 0x03030303, 0x03030303,
+    0x03030303, 0x03030302, 0x01010101, 0x02020202, 0x02020202, 0x02020303, 0x03030303, 0x03030303, 0x02010100,
+    0x00000102, 0x03030302, 0x02010102, 0x02020100, 0x00000000, 0x00000000,
+};
+
+static UNK_TYPE D_80B69660[] = {
+    0xFC18FE6B, 0x00000000, 0x00000000, 0xFFFFFFFF, 0x0000FE6B, 0x00000000, 0x00000000, 0xFFFFFFFF,
+    0x03E8FE6B, 0x00000000, 0x00000000, 0xFFFFFFFF, 0xFE0C0195, 0x00000000, 0x00000000, 0xFFFFFFFF,
+    0x01F40195, 0x00000000, 0x00000000, 0xFFFFFFFF, 0x00000500, 0x00000000, 0x00000000, 0xFFFFFFFF,
+};
+
+static Gfx triforceDL[] = {
+    gsDPPipeSync(),
+    gsDPSetTextureLUT(G_TT_NONE),
+    gsSPTexture(0xFFFF, 0xFFFF, 0, G_TX_RENDERTILE, G_OFF),
+    gsDPSetCombineLERP(0, 0, 0, PRIMITIVE, 0, 0, 0, 1, 0, 0, 0, COMBINED, 0, 0, 0, COMBINED),
+    gsSPSetOtherMode(G_SETOTHERMODE_L, G_MDSFT_RENDERMODE, 29, 0xC1849D8),
```

Comment:
Are all these display lists generated from `gfxdis`? They don't look quite right.

Reply:
Some of it was from random's utility. The only things i noticed was that i could change gsSPSetOtherMode to gsDPSetRenderMode. Also a few minor changes using actual names instead of 0